t/mime.rakutest

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env raku
+
```

Comment:
To tell you the truth, this is the first time I've seen this extension and I'm not totally sure it's canon. That might explain why tests have failed...

Reply:
It's certainly the longest file extension hats for sure! I got the naming from here:

https://github.com/Raku/problem-solving/blob/master/solutions/language/Path-to-Raku.md#extensions

But happy to hear if there has / is a change.

Reply:
Well, there you go. Tests worked; they failed because they were trying to use panda...